nexus/catalog/migrations/V28__ch_s3_stage_unique_constraint.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE ch_s3_stage ADD CONSTRAINT 
```

Comment:
merge the migrations in one file

Reply:
testing locally was a pain as i have to reset catalog etc :-/